ddtrace/contrib/django/patch.py

```diff
@@ -90,7 +91,17 @@ def cursor(django, pin, func, instance, args, kwargs):
             and isinstance(cursor.cursor, psycopg_cursor_cls)
         ):
             traced_cursor_cls = Psycopg2TracedCursor
-        return traced_cursor_cls(cursor, pin, config.django)
+        # Each db alias will need its own config for dbapi
+        cfg = IntegrationConfig(
+            config.django.global_config,  # global_config needed for analytics sample rate
+            "{}-{}".format("django", alias),  # name not used but set anyway
+            _default_service=config.django._default_service,
+            _dbapi_span_name_prefix=prefix,
+            trace_fetch_methods=config.django.trace_fetch_methods,
```

Comment:
Worth noting here that we'll lose the ability to dynamically change these configurations since they're copies of the original django config now